granary/reddit.py

```diff
@@ -230,6 +229,21 @@ def get_activities_response(self, user_id=None, group_id=None, app_id=None,
 
     return self.make_activities_base_response(activities)
 
+  def get_comment(self, comment_id, activity_id=None, activity_author_id=None,
+                  activity=None):
+    """Returns an ActivityStreams comment object.
+
+    Args:
+      comment_id: string comment id
+      activity_id: string activity id, Ignored
+      activity_author_id: string activity author id. Ignored.
+      activity: activity object, Ignored
+    """
+    r = get_reddit_api(self.refresh_token)
```

Comment:
doesn't have to be in this PR, but i wonder if we should only instantiate `praw.Reddit` once, and keep it in the class. instantiating it probably makes at least one API call, to get an access token, if not more, right? we don't really want to be doing that every time we call one of these methods.

Reply:
I was wondering about this, but I thought that since the access token expires after 60 minutes maybe it could cause problems if the praw object is stored for too long.

Reply:
eh. if it's using the refresh token to get an access token once, i expect it knows to look at expiration and get another one if necessary? could look.

more importantly for bridgy at least, instances of these granary classes only live for an HTTP request, which even for very long polls generally isn't more than 30-60s. other granary users could be longer though.

Reply:
I made a new issue to follow up on this later.  https://github.com/snarfed/granary/issues/202